### PR TITLE
refactor: BlockExplorerLink component

### LINKED ISSUE #3298: BlockExplorerLink - new component
- would be cool to create a new component `BlockExplorerLink` in the future
- create new component 

_Originally posted by @roiLeo in https://github.com/kodadot/nft-gallery/pull/3297#pullrequestreview-1021792389_

edit: 

we are looking for ways to make the app more composable and flexible, right now, many parts of the BSX implementation are copied from the existing rmrk paths. We would like to have separate component which would handle links to blocks / timestamps in block explorer (such as subscan)

check for reference: 
- #3280
- #3297

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3298 
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

RMRK:
history\holder\filppers...
image

BSX:
history\holder\filppers...
image

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "lib": ["ESNext", "ESNext.AsyncIterable", "DOM"],
+    "jsx": "react",
```
Comment (roiLeo) [inline tsconfig.json:7]: What's this for?

**Review by roiLeo — APPROVED** [review]
Nice refactoring thank you
✅ lgtm

File: tsconfig.json (hunk 1 above)
Comment (roiLeo) [reply tsconfig.json:7]: Am I missing something to setup? Cause I don't have this error

Capture d’écran 2022-06-30 à 3 16 13 PM

Comment (roiLeo) [reply tsconfig.json:7]: That's weird it's not even jsx, well let's keep it that way

**Comment by roiLeo** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 45.99 USD/KSM ~ 1.087 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x77fc597b0732c11c5adca231363f34c8088993167eaebca4dc084c4faf6ddc5b](https://kusama.subscan.io/extrinsic/0x77fc597b0732c11c5adca231363f34c8088993167eaebca4dc084c4faf6ddc5b)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

File: tsconfig.json (hunk 1 above)
Comment (Jarsen136) [reply tsconfig.json:7]: OK

**Pull request merged**